src/dotnet/ExecutablePackageObtainer/ExecutablePackageObtainer.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using Microsoft.Extensions.EnvironmentAbstractions;
+
+namespace Microsoft.DotNet.ExecutablePackageObtainer
+{
+    public class ExecutablePackageObtainer
```

Comment:
I see you added InternalsVisibleTo but also made everything public. Why?

Reply:
Marked all to internal. This is an oversight